### PR TITLE
Incorporate BHarsh feedback on 2.0-compatible CLBG entries

### PR DESCRIPTION
**Pull request by bradcray**
This PR makes three changes in response to feedback from BHarsh upon his post-merge review of the CLBG edits I've been making this week:

* the first, and simplest, is to change a mulit-var declaration in revcomp8.chpl back to two distinct declarations.  This reflects his justifiable and preferred style (adjacent but unrelated declarations shouldn't necessarily share a `var` keyword) and also represents a case of over optimization on my part since this is a super-long and complex benchmark (i.e., we weren't going to be saving anything noticeable in terms of code size over this).

* the second is to change the readLine() to use `stripNewline=true` since that's what we want to do anyway.  This removes the need for a `+1` adjustment I'd had to make to the domain size in order to compensate for the new readLine() routine going around the loop an extra time compared to the previously submitted version, resulting in an extra subtraction of 1 within the loop body (the old readline() used to test for and return EOF, preventing this extra trip around the loop).  This version seems ideal in that, rather than manually dancing around and overwriting linefeeds, we just ask the routine to discard them for us.  Nice!

* finally, while looking into the previous bullet, I learned that the problem size currently used to verify knucleotide on the CLBG site includes some map lookups for strings that are not in the map, which resulted in OOB errors.  I believe the problem size we've been testing against was the old verification problem size, and ensured that all strings were in the map.  Here, I've changed from indexing to the sentinel-taking `.get()` method to handle such cases and added new input/output sets matching those on the site today.

### CODE REVIEW

File: test/studies/shootout/submitted/knucleotide-output.txt.good [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+TG 6.201
+GT 6.041
+CT 5.961
+AG 5.841
+CA 5.461
+AC 5.441
+CC 4.041
+CG 4.021
+GC 3.701
+GG 3.341
```
Comment (benharsh) [inline test/studies/shootout/submitted/knucleotide-output.txt.good:21]: Have you checked this PR with the .perfkeys verification values?

Comment (bradcray) [reply test/studies/shootout/submitted/knucleotide-output.txt.good:21]: Good thought, I haven't.  You're not expecting it to break, though, are you?  (in that the behavior for get() should be the same when there are no missing keys?

Comment (benharsh) [reply test/studies/shootout/submitted/knucleotide-output.txt.good:21]: It shouldn't, but after our chat with the diff:

```
< C 19.773
---
> C 19.772

< TC 5.958
---
> TC 5.957
```

I may be a bit paranoid. Especially in the interest of getting performance results tonight.

Comment (bradcray) [reply test/studies/shootout/submitted/knucleotide-output.txt.good:21]: Looks clean as expected.

**Review by benharsh — APPROVED** [review]

**Review by benharsh — APPROVED** [review]

**Pull request merged**